README.md

```diff
@@ -61,12 +88,67 @@ mount OmbuLabs::Auth::Engine, at: '/', as: 'ombu_labs_auth'
 </div>
 ```
 
+> This will default to a basic HTML page included in this gem. To customize this view, check [this section](#customizing-sign-in-page)
+
 - Add the Devise authentication helper to your private objects controllers
 
-```
+```rb
 before_action :authenticate_user!
 ```
 
+- Include the `OmbuLabsAuthenticable` concern in the authenticable model
+
+```rb
+class Admin < ApplicationRecord
+  include OmbuLabsAuthenticable
+  ...
+end
+```
+
+- Tell `OmbuLabs::Auth` the user class name and table for the authenticable model
+
+```rb
+# config/initializers/ombu_labs-auth.rb
+OmbuLabs::Auth.user_class = "Admin" # defaults to "User" if not set
```

Comment:
@arielj I might be missing something but I don't see a setter method in module `OmbuLabs::Auth` -- did you mean to say this? 

```suggestion
OmbuLabs::Auth.user_class_name = "Admin" # defaults to "User" if not set
```